src/components/Modals/centerModals/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'react';
+import { TouchableOpacity, View } from 'react-native';
+import DatePicker from 'react-native-modern-datepicker';
+import { useSelector } from 'react-redux';
+import { getColors } from 'src/store/selectors';
+import { RED_200, WHITE_100 } from 'src/styles';
+
+import { styles } from './DatePicker.style';
+import { getFormattedDate } from './DatePicker.utils';
+import Text from '../../../Text/Text';
+import ModalContainer from '../../ModalContainer/ModalContainer';
+
+type Props = {
+  label?: string;
+  width?: number;
+  onDateChange: (selectedDate: string) => void;
+  error?: string;
+  initialValue?: string;
+};
+
+const DateTimePicker = ({ label, width, onDateChange, error, initialValue }: Props) => {
+  const colors = useSelector(getColors);
+  const initialDate = initialValue
+    ? new Date(initialValue).toISOString().split('T')[0].replace(/-/g, '/') // yyyy-mm-dd
```

Comment:
make separate file like `formatters` on common utils folder and better use dayjs